mysql-test/suite/sys_vars/t/allow_suspicious_udfs.test

```diff
@@ -0,0 +1,12 @@
+#
+# MDEV-24815
+# Show allow-suspicious-udfs state in SYSTEM VARIABLES
+#
```

Comment:
This test isn't restarting the same state. Recommend:

Use a `allow_suspicious_udfs.opt` containing the server option `allow-suspicious-udfs`. Move the `EXPECT_0` case to a different test file (something that tests default values already).

or b) restart these server again at the end with `--skip-allow-suspicious-udfs`

This might be the cause of the mysql_upgrade-6984 test case.

Reply:
Done